astropy/coordinates/transformations.py

```diff
@@ -347,23 +347,38 @@ def get_transform(self, fromsys, tosys):
             self._composite_cache[fttuple] = comptrans
         return self._composite_cache[fttuple]
 
-    def lookup_name(self, name):
+    def lookup_name(self, name, suppress_error=True):
         """
         Tries to locate the coordinate class with the provided alias.
 
         Parameters
         ----------
         name : str
             The alias to look up.
+        suppress_error : bool (default value is False)
+            True : doesn't raise exception if 'name' isn't found. (returns None)
+            False : raises exception if 'name' isn't found.
```

Comment:
Could you also add a `.. versionadded:: 3.0` to the parameter description?

And I think it would be better as a sentence like `If True (the default) then it returns None if the name isn't found otherwise it will raise an Exception`. Sphinx generally doesn't respect newlines so this could look weird. You may need to build the documentation locally to see if it looks alright.